src/util/fs.js

```diff
@@ -144,21 +144,24 @@ async function buildActionsForCopy(
       srcFiles = await readdir(src);
     }
 
-    if (await exists(dest)) {
-      const destStat = await lstat(dest);
+    let destStat;
+    try {
+      destStat = await lstat(dest);
+    } catch (e) {}
```

Comment:
there needs to be an explanation for a muted exception.
Could you add a one line comment?